RELEASE_NOTES.md

```diff
@@ -1,153 +1,136 @@
-# Chef Client Release Notes 12.1.0:
-
-# Internal API Changes in this Release
-
-## Experimental Audit Mode Feature
-
-This is a new feature intended to provide _infrastructure audits_.  Chef already allows you to configure your infrastructure
-with code, but there are some use cases that are not covered by resource convergence.  What if you want to check that
-the application Chef just installed is functioning correctly?  If it provides a status page an audit can check this
-and validate that the application has database connectivity.
-
-Audits are performed by leveraging [Serverspec](http://serverspec.org/) and [RSpec](https://relishapp.com/rspec) on the
-node.  As such the syntax is very similar to a normal RSpec spec.
-
-### Syntax
-
-```ruby
-control_group "Database Audit" do
-
-  control "postgres package" do
-    it "should not be installed" do
-      expect(package("postgresql")).to_not be_installed
-    end
-  end
-
-  let(:p) { port(111) }
-  control p do
-    it "has nothing listening" do
-      expect(p).to_not be_listening
-    end
-  end
-
-end
+# Chef Client Release Notes 12.2.0:
+
+## Desired State Configuration (DSC) Resource
+
+If you are using `WMF 5`, you can now take advantage of the new `dsc_resource`. 
+This new functionality takes advantage of WMF 5's `Invoke-DscResource` cmdlet to
+directly invoke resources.
+
+### Prerequisites
+
+To use this new resource, you must have the February preview of WMF 5.
+This can be installed using the Powershell cookbook. It is also required that
+the Local Configuration Manager(LCM) be configured with a `RefreshMode` of `Disabled`.
+Doing this will preclude you from using `dsc_script`. Below we provide an example
+DSC configuration:
+
+```powershell
+Configuration LCMSettings {
+  Node localhost {
+    LocalConfigurationManager {
+      RefreshMode = 'Disabled'
+    }
+  }
+}
```

Comment:
We probably want to finish this example with how to run the configuration, since folks who prefer the dsc_resource method may not be as familiar with DSC itself.

```powershell
# create a configuration command to generate a meta.mof to set Local Configuration Manager settings
Configuration LCMSettings {
  Node localhost {
    LocalConfigurationManager {
      RefreshMode = 'Disabled'
    }
  }
}
# Run the configuration command and generate the meta.mof to configure a local configuration manager
LCMSettings
# Apply the local configuration manager settings found in the LCMSettings folder (by default configurations are generated 
# to a folder in the current working directory named for the configuration command name
Set-DscLocalConfigurationManager -path ./LCMSettings
```